wzdx/experimental_combination/attenuator.py

```diff
@@ -36,6 +40,30 @@ def main(outputPath="./tests/data/output/wzdx_attenuator_combined.json"):
         logging.debug(f"Output written to {outputPath}")
 
 
+# parse script command line arguments
+def parse_rtdh_arguments():
+    parser = argparse.ArgumentParser(
+        description="Combine WZDx and Geotab AVL (ATMA) data"
+    )
+    parser.add_argument(
+        "--version", action="version", version=f"{PROGRAM_NAME} {PROGRAM_VERSION}"
+    )
+    parser.add_argument("wzdxFile", help="planned event file path")
+    parser.add_argument("geotabFile", help="planned event file path")
+    parser.add_argument(
+        "--outputDir", required=False, default="./", help="output directory"
+    )
+    parser.add_argument(
+        "--updateDates",
+        required=False,
+        default="false",
+        help="Boolean (true/false), Update dates to the current date to pass time filter",
+    )
+
+    args = parser.parse_args()
```

Comment:
With this update do we need to also update our corresponding README documentation? This should be expecting filenames as arguments now but our README still mentions just running `python attenuator.py`

Reply:
Updated!